server.js

```diff
@@ -1,25 +1,29 @@
-const express = require("express");
-const bodyParser = require("body-parser");
-const cors = require("cors");
+const express = require('express');
+const cors = require('cors');
 
-const { handleSignin } = require("./controllers/signin");
-const { handleSignout } = require("./controllers/signout");
-const { requireAuth } = require("./controllers/authorization");
-const { getLayout } = require("./controllers/drumLayout");
+const { handleSignin } = require('./controllers/signin');
```

Comment:
- [ ] Can you change all of the single quotes back to double quotes? The project standard is to use double quotes before single quotes :)

Reply:
@rgavinc Fixed.